core/templates/pages/profile-page/profile-page.component.html

```diff
@@ -26,20 +26,13 @@
       </div>
 
       <div class="oppia-profile-text">
-        <h2 class="oppia-profile-username-large-screen">
+        <h1 class="oppia-profile-username-large-screen oppia-profile-username-small-screen">
```

Comment:
Can you please this attribute --  `aria-label="Username"` to make it more accessible, also this change can help you fix the failing lighthouse test.

Reply:
@Patel-Muhammad i think aria-label should be like aria-label="user name"